public_dropin_gpu_environments/python311_genai/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cloudpickle==2.2.1
+torch==2.0.1
+transformers==4.39.3
+openai==1.30.2
+langchain==0.1.20
+optimum==1.19.2
+onnxruntime==1.17.3
+onnx==1.16.0
+sentence-transformers==3.0.0
+faiss-cpu==1.8.0
```

Comment:
Not sure about this. Should we try to use faiss-gpu my understanding that there restriction about what cuda version it supports.

Reply:
Now it's my turn :)
 I see you put `custom-models` as owner. I disagree :) Please make it buzok

Reply:
Custom Models can not be an owner, we know nothing about this env.
`python311_genai` will likelyl be owned by @baekdahl

Reply:
Yes, I can own this.
@mkavf What is the point of this environment? To me, it looks identical to the non-gpu python311_genai.

Reply:
@baekdahl 

It is almost identical. In Docketfile I changed `https://download.pytorch.org/whl/cpu` in regular python311_genai to `https://download.pytorch.org/whl/cu118`. This will allow us to utilize  GPU's.

Also I considering changing `faiss-cpu` to `faiss-gpu` but now sure what version of CUDA we support or should support

Reply:
Seem there no corresponding version of  `faiss-gpu` for our version, so i will leave `faiss-cpu` as is.